### ISSUE TITLE
[Feature Request]: Support "Videos" Plex Libraries 

### ISSUE THREAD

**Issue by treysu** [tubearchivist/tubearchivist-plex#88]
### I've read the documentation.

- [X] I have read the [current limitations](https://github.com/tubearchivist/tubearchivist-plex#limitations), and confirmed that this hasn't been requested as a [feature request](https://github.com/tubearchivist/tubearchivist-plex/issues?q=is%3Aissue) previously.

## Is your feature request related to a problem? Please describe.

I often download one-off videos and I don't want them sorted by channel. Sometimes they are random uploads of other things I want to save, sometimes they are just single videos.

## Describe the solution you'd like

I would like to be able to use the "Videos" type library instead of the "TV Shows" library.

## Additional context

I am happy to work on this with my Python knowledge but I don't know exactly how this scanner importing works. I have tried messing with a custom scanner for Video libraries, but I haven't gotten it working yet.

### Your help is needed!

- [x] Yes, I will work on this in the next few days or weeks.

**Comment by lamusmaser** [comment]
Because of how this specific utility interacts with Plex, the TV Series type is better than the Other Videos type. I attempted to use it at the beginning, but it does not support many mechanisms that are required for other areas.

We could have it be its own thing, where if someone does select the Other Videos type, it will just supply the videos, but that would require major updates to both the Scanner and Agent to ensure those details are being pulled in the format the Plex expects for that type.

If you have additional details, I can consider it, but otherwise, I will say that this is something I don't currently have time to work on, so this would be behind all of the other Feature Requests.

**Comment by treysu** [comment]
Firstly, thank you for taking the time to read and consider my feature request. I am very grateful for the work done on this project and appreciate all you do. 

If we are stuck to one format, I can definitely see the reason why TV Shows was chosen as the primary solution, but I was hoping it would simple enough to provide an alternate Scanner/Agent pair to support Other Videos. Looking at the code it doesn't seems crazy complex, but it sounds like there are differences behind the scenes between TV Show libraries and Other Videos libraries that might make it more complicated. 

I completely understand that this is beyond a standard feature request, and I don't expect any direct development to take place at this time. I am extremely interested in contributing to this solution myself, and I do have a good amount of Python experience, although I know very little about the Plex scanners/agents at this point. I also have seen that Plex plans to [sunset the use of Python scanners and agents](https://forums.plex.tv/t/want-to-create-a-custom-scanner/776312), so I can understand if there is little interest in learning a new side of the technology. 

If you have any specific resources or individuals that might be able to help me get started on Plex Agent/Scanner development, I would greatly appreciate it. In the meantime, I would appreciate this being left **open** as a **low priority** feature request, just in case anyone else sees this and is interested in development work.

**Comment by lamusmaser** [comment]
Absolutely, feel free to contribute. As for resources, they are slim to none - this project was built by reverse engineering other projects and the core code. 

I will take the deprecation notice with a grain of salt since I've been seeing reports about that for years and years. If they do, then it happens. Otherwise, we have something that works in the meantime. 

As for my recommendation, I would go look at the code for Other Videos Scanner and see if that can help. It might be easy enough to call different methods based on setting, as other Scanners have done so.

**Comment by lamusmaser** [comment]
Because of the methods required, if I look into this further, it will need to come into v0.2.0 or later, since that may involve some refactoring of certain elements of the code base anyway.
